slice/IceGrid/Admin.ice

```diff
@@ -208,6 +208,76 @@ module IceGrid
         ApplicationUpdateDescriptor descriptor;
     }
 
```

Comment:
These exceptions were copy-pasted (with no changes) from `Exception.ice`.
`Admin.ice` is the only Slice file where they're used, and moving them was necessary for some `@see` in `Admin.ice` to link properly.